src/patient-registration/patient-registration.component.tsx

```diff
@@ -503,6 +504,17 @@ export const PatientRegistration: React.FC = () => {
           results.then(response => {}).catch(err => {});
 
           navigate({ to: getAfterUrl(response.data.uuid) });
+          existingPatient
+            ? showToast({
+                description: "The patient's information has been successfully updated",
```

Comment:
Is this taken from the design?

If yes, ok - but we need all these strings translated. If no - I'd argue to drop it.

Reply:
Yes it is also from the designs, I will get it translated.